examples/machine/DeePMD-kit-2.x/lebesgue_v2_machine.json

```diff
@@ -0,0 +1,107 @@
+{
+  "api_version": "1.0",
+  "train" :[
+    {
+      "command": "dp",
+      "machine": {
+        "batch_type": "DpCloudServer",
+        "context_type": "DpCloudServerContext",
+        "local_root" : "./",
+        "remote_profile":{
+          "email": "your lebesgue register email",
+          "password": "your lebesgue password",
+          "program_id": 0,
+            "input_data":{
+                "api_version":2,
```

Comment:
There is lack of description for `api_version` in `input_data`.One doesn't know the difference between this and `api_version` outside.

Reply:
thank you for your response, I have added the description about "api_version" in the README file.